tekton/src/params/common.yml.j2

```diff
@@ -15,6 +15,10 @@
   default: ""
   type: string
   description: Required to install development MAS catalogs
+- name: dev_mode_airgap
```

Comment:
See previous comment re: whether we can avoid the need to pass this to the pipeline.